test/linter/test-versions.js

```diff
@@ -89,6 +94,12 @@ function checkVersions(supportData, relPath, logger) {
             }
           }
         }
+        if ('flags' in statement) {
+          if (FLAGLESS_BROWSERS.indexOf(browser) != -1) {
```

Comment:
A little less cryptic, hopefully

```suggestion
          if (FLAGLESS_BROWSERS.includes(browser)) {
```